pages/design/tools/figma.mdx

```diff
@@ -1,6 +1,110 @@
+import {Flex, Box, Text, BorderBox} from '@primer/components'
+
 # Figma
+Welcome to the Figma page of the Primer design system! Figma is used at GitHub as a single source of design, collaboration, and exploration.
+
+<Text textAlign="center">
+   <Flex alignItems='center' flexDirection='row' mt='4' mb='4'>
+    <Box width={12/12}>
+      <BorderBox height="250px" overflow='hidden'>
+       <img src="https://user-images.githubusercontent.com/6846673/61730055-6ddf6000-ad2d-11e9-92ce-cd14fccc6851.png"/>
+      </BorderBox>
+    </Box>
+   </Flex>
+</Text>
+
+This page can help you learn how to use:
+* [Team Library](#Library)
+* [Styles](#Styles)
+* [Components](#Components)
+* [Prototyping and Wireframing](#Prototyping)
+* [Collaboration](#Collaboration)
+
+## <a name="Library"></a>Team Library
+GitHub has a Figma team that contains all of GitHub's design assets. In Figma, [styles](#Styles) and [components](#Components) can be reused across multiple files and projects with the [team library](https://help.figma.com/article/29-team-library) feature. With team libraries, you can enable and disable components styles within certain files. This makes it easy for designers to easily access Primer components from anywhere.
+
+<Text textAlign="center" fontStyle='italic'>
+   <Flex alignItems='center' flexDirection='row' mt='4' mb='4'>
+    <Box width={12/12}>
+      <BorderBox overflow='hidden'>
+       <img width="100%" src="https://user-images.githubusercontent.com/6846673/61732332-3f17b880-ad32-11e9-8e18-bfbfc0267d81.gif"/>
+      </BorderBox>
+      <Text textAlign='center' color='gray.6'>Enable and disable libraries of components from across your team.</Text>
+    </Box>
+   </Flex>
+</Text>
+
+## <a name="Styles"></a>Styles
+Primer uses Figma styles to maintain a consistent look and feel across all of GitHub's UI. We have provided styles for color variables as well as our typography.
```

Comment:
should link to the Figma styles resource here? "Primer uses [Figma styles](#).."